### DISCUSSION TITLE
Unforseen problems with legalities of information

### DISCUSSION THREAD

**Discussion by Nicroe69** [community/community#146472] [Actions]
So I've been trying to get this stuff figured out . And to be, it's just kind of mind-blowing. the abilities that a person has to access our information is astounding. I've been on this Quest for nearly 3 year's,  in search of the truth. I've been on the legislature sites on legal sites law schools sites reference a lot a lot of law, to ultimately find out the truth that I seek, is knowledge base and is controlled,  by entities you don't want to upset. My question is, how do you know if you have a document or you shared a document that maybe you weren't supposed to have or possibly weren't supposed to share with somebody? To my knowledge that I say this wholeheartedly and in  full faith, I have done nothing wrong as the information I've sought is only been on myself. How does one go about assessing the help needed for further understanding? I do know that I uncovered within the law realm has been rather serious and does pertain to everyone as far as knowledge base.  I'm certain that knowledge is a misdirection and understandable ability to sort fact from fiction. However, it's their laws... I just know how to read them. And there are far too many for any one person could survive in everyone ever written.  I'm interested in some very old ones,  and not really discussed respectively. Therefore, it's takes a certain amount of knowledge to be able to know where to look.  Since I have discovered it,  they have been relocated and hidden again naturally. But I was on a public accessible site when I viewed it.  Did anything ever happen similarly to anyone else? I'm asking for a friend...

**Comment by AnilChettri** [comment]
You sound careful and conscientious — that’s exactly the right mindset. Don’t panic, don’t overshare, and don’t try to “prove” anything by poking more. Document how you found the material, report it responsibly, and if the content is high-risk (classified or could cause harm), get a lawyer before doing anything else. If you want, I’ll draft a message to the site admin or a short note you can show to an attorney — say which you prefer and I’ll write it now.
